packs/equipment/belt-of-good-health.json

```
@@ -3,31 +3,13 @@
     "img": "icons/equipment/waist/belt-coiled-leather-steel.webp",
     "name": "Belt of Good Health",
     "system": {
-        "MAP": {
-            "value": ""
-        },
-        "ability": {
-            "value": "str"
-        },
         "baseItem": null,
-        "bonus": {
-            "value": 0
-        },
-        "bonusDamage": {
-            "value": 0
-        },
         "bulk": {
             "value": 0
         },
         "containerId": null,
-        "damage": {
-            "damageType": "slashing",
-            "dice": 1,
-            "die": "d6",
-            "value": ""
-        },
         "description": {
-            "value": "<p>When you put on this belt, its silver buckle begins to glow, which slowly spreads into the heart-shaped jewel in the center. You increase your maximum Hit Points and current Hit Points by 4. If you remove the belt, you immediately decrease both your maximum and current HP by 4.</p>"
+            "value": "<p><span style=\"font-family:Roboto, sans-serif\">When you put on this belt, its silver buckle begins to glow, which slowly spreads into the heart-shaped jewel in the center. You increase your maximum Hit Points and current Hit Points by 4. If you remove the belt, you immediately decrease both your maximum and current HP by 4.</span></p>"
```

```suggestion
            "value": "<p>When you put on this belt, its silver buckle begins to glow, which slowly spreads into the heart-shaped jewel in the center. You increase your maximum Hit Points and current Hit Points by 4. If you remove the belt, you immediately decrease both your maximum and current HP by 4.</p>"
```
Redundant html snuck in.